WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -660,6 +725,62 @@ class PostListViewModel @Inject constructor(
         }
     }
 
+    private fun loadPostFromRemote(localPostId: Int) {
+        // We need network connection to load a remote post
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        val post = postStore.getPostByLocalPostId(localPostId)
+        if (post != null) {
+            originalPostCopyForConflictUndo = post.clone()
+            dispatcher.dispatch(PostActionBuilder.newFetchPostAction(RemotePostPayload(post, site)))
+            _toastMessage.postValue(ToastMessageHolder(R.string.toast_conflict_updating_post, Duration.SHORT))
+        } else {
+            localPostIdForConflictResolutionDialog = null
+        }
+    }
+
+    private fun savePostFromLocal(localPostId: Int) {
+        // We need network connection to push local version to remote
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        val post = postStore.getPostByLocalPostId(localPostId)
```

Comment:
```suggestion
        val post = postStore.getPostByLocalPostId(localPostId) ?: return
```

We can simply do this ^ and remove the extra `null` check below.

```
if (post == null) {
    return
}
```